install-launchd.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/sh
+
+set -e
+
+if [ "$UID" -eq 0 ] ; then
+    echo "$0 is not meant to be run as root" >&2
+    exit 1
+fi
+
+calcheck="$HOME/Library/LaunchAgents/com.calcheck.timer.plist"
+dir=$(pwd)
+stdout="$dir/log.out"
```

Comment:
logging to the source directory is pretty weird... is there a more natural place to put logs?

Also... is it maybe possible to let launchctl configure log locations itself?  I don't know much about launchctl but I'd expect that from any daemon manager system.

Reply:
yeah, i agree. launchctl isn't great though. for whatever reason i could not get the echo outputs/print statements to show up in /var/sys/ (which is the default path when there's nothing configured). this issue is basically same as mine: https://apple.stackexchange.com/questions/108717/launchd-scripts-output-not-being-logged-to-system-log and the suggestion was to just log to your own log files. i'm open to other directories other than source, but there will be some permissions issues to figure out, since launchctl runs as not-root.

Reply:
for a user-based service, as you are mandating here, I'd expect `/var/sys` to be inaccessible, and for logs to appear somewhere in `$HOME`.  In linux, it would be in `$HOME/.local/var/sys` or something

Reply:
Can you try running it with `some_unique_key_name.log`, then do a `sudo find $HOME -name some_unique_key_name.log` to see if it's being written somewhere unexpected?

Reply:
without specifying an output, i can't find any logs in my home directory. no idea where the logs go. i think what i can do is create log files in the home directory and give them user permissions needed. so where should i put them inside the home directory?

Reply:
nah, it's not worth it.  I'll stamp what you've got.